### PR TITLE
feat: add license agent openapi proxy

### PR DESCRIPTION
**Pull request by CeerDecy**
#### What this PR does / why we need it:

add license-agent openapi proxy

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @iutx @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | add license agent openapi proxy |
| 🇨🇳 中文    | 添加了对license-agent的openapi转发 |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: internal/core/openapi/legacy/api/apis/license_agent/agent_license_feature_get.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package license_agent
+
+import (
+	"github.com/erda-project/erda/internal/core/openapi/legacy/api/apis"
```
Comment (iutx) [inline internal/core/openapi/legacy/api/apis/license_agent/agent_license_feature_get.go:4]: Please modify the Go import order for all Go files.

File: internal/core/openapi/legacy/api/apis/license_agent/agent_license_feature_get.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package license_agent
```
Comment (iutx) [inline internal/core/openapi/legacy/api/apis/license_agent/agent_license_feature_get.go:1]: Please identify the license for the project.

File: go.sum [hunk 3]
```diff
@@ -38,6 +38,8 @@ cloud.google.com/go v0.100.2/go.mod h1:4Xra9TjzAeYHrl5+oeLlzbM2k3mjVhZh4UqTZ//w9
 cloud.google.com/go v0.102.0/go.mod h1:oWcCzKlqJ5zgHQt9YsaeTY9KzIvjyy0ArmiBUgpQ+nc=
```
Comment (iutx) [inline go.sum:38]: If you did not change the go.mod file, please avoid committing the go.sum content.

**Review by iutx — COMMENTED** [review]
@CeerDecy

**Review by iutx — APPROVED** [review]
LGTM @sfwn

File: .erda/migrations/license-agent/20230927-license-base.sql [hunk 4]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE erda_licenses
```
Comment (sfwn) [inline .erda/migrations/license-agent/20230927-license-base.sql:1]: use singular format for table name: `erda_license`

File: internal/core/openapi/legacy/api/apis/license_agent/agent_license_feature_get.go [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+import (
+	"net/http"
+
+	"github.com/erda-project/erda/internal/core/openapi/legacy/api/apis"
+)
+
+var AGENT_LICENSE_FEATURE_GET = apis.ApiSpec{
+	Path:        "/api/licenses/features",
```
Comment (sfwn) [inline internal/core/openapi/legacy/api/apis/license_agent/agent_license_feature_get.go:24]: strange path

**Review by sfwn — CHANGES_REQUESTED** [review]

File: internal/core/openapi/legacy/api/apis/license_agent/agent_license_feature_get.go (hunk 5 above)
Comment (CeerDecy) [reply internal/core/openapi/legacy/api/apis/license_agent/agent_license_feature_get.go:24]: how about `/api/licenses/obtain` ?

Comment (CeerDecy) [reply internal/core/openapi/legacy/api/apis/license_agent/agent_license_feature_get.go:24]: image

**Review by sfwn — APPROVED** [review]

**Pull request merged**